src/controllers/articles.controller.js

```diff
@@ -62,7 +63,8 @@ class Articles {
           favoritedcount: createdArticle.favoritedcount,
           images: createdArticle.images,
           createdAt: createdArticle.createdAt,
-          updatedAt: createdArticle.updatedAt
+          updatedAt: createdArticle.updatedAt,
+          taglist: createdArticle.taglist
```

Comment:
I think there is a tag list table where they have to be saved, I suggest making use of it.

Reply:
I was requested to add taglist as I create articles and then, save articles along with their tags. Thank you for the feedback

Reply:
in addition I can't save data from front-end indifferent tables